packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -797,6 +799,19 @@ const ruleset: any = {
         function: noDuplicatePathsForScopeParameter,
       },
     },
+    // RPC Code: RPC-Uri-V1-11
+    ExceptionMandateForTenantLevelApiPath: {
+      description: "Exception from PAS team is mandatory for Tenant level PUT operation.",
```

Comment:
>Exception from PAS team is mandatory for Tenant level PUT operation. [](http://example.com/codeflow?start=20&length=68)

hmm lets shift focus away from the exception to the actual rule we want to enforce. 

So call it TenantLevelAPIsNotAllowed